README.md

```diff
@@ -17,20 +17,17 @@ Hyprland's simple, yet multi-threaded and GPU-accelerated screen locking utility
 ## Building
 
 ### Deps
-Hyprlock requires `hyprlang` >= 0.4.0. See the 
-[README](https://github.com/hyprwm/hyprlang/blob/main/README.md#building-and-installation) for 
-build instructions.
-
 You also need the following dependencies
- - wayland-client
- - wayland-protocols
- - mesa
- And the development libraries for the following:-
- - cairo
- - libdrm
- - pango
- - xkbcommon
- - pam
+- wayland-client
+- wayland-protocols
+- mesa
+And the development libraries for the following:-
```

Comment:
why is there a - at the end here xd

Reply:
I just like to add a - in these types of lines, just have a habit of it. I will remove that if you don't like that.

Reply:
ye please dont